src/babel/transformation/transformers/index.js

```diff
@@ -1,124 +1,66 @@
 export default {
   "utility.removeDebugger":                require("./utility/remove-debugger"),
   "utility.removeConsole":                 require("./utility/remove-console"),
-
```

Comment:
Whitespace and comments were removed because the comments were outdated and the whitespace separating transformers was pretty arbitrary and gave the impression that it was somehow significant/related when it wasn't in the slightest.